### PR TITLE
Add OKR methodology

### PR DESCRIPTION
**Pull request by gomesfernanda**
Signed-off-by: Fernanda Gomes <fergomes@gmail.com>

### CODE REVIEW

**Review by smola — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,135 +1,12 @@
-# OKRs 2019 Q1
+# source{d} OKRs
 
-## Bring 'Code as Data' to market with enterprises
+## Methodology
 
-### Enable engine integration with 3rd parties
+- [source{d} OKR methodology](https://github.com/src-d/okrs/blob/master/2018-Q1.md)
```
Comment (carlosms) [inline README.md]: wrong link?
```suggestion
- [source{d} OKR methodology](https://github.com/src-d/okrs/blob/master/sourced-okr-methodology.md)
```

File: sourced-okr-methodology.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+#### Most common mistakes
+
+* Setting objectives that are not ambitious enough.
+* Not having a clear understanding of the challenge you want to solve.
+* Having more than 3 Key Results per Objective.
+* Not involving all the stakeholders on OKRs elaboration.
+* Not communicating properly the process throughout the organization.
+* Writing poor OKRs (that don't measure customer behavior, but measures features delivery).
+* Writing OKrs that fail to scale (not tying into each other and not making sense strategically throughout the organization).
```
Comment (dpordomingo) [inline sourced-okr-methodology.md]:
```suggestion
* Writing OKRs that fail to scale (not tying into each other and not making sense strategically throughout the organization).
```

**Review by dpordomingo — APPROVED** [review]

**Review by marnovo — APPROVED** [review]
LGTM (had already written the bulk and reviewed the PR-ed version).

**Comment by marnovo** [comment]
@gomesfernanda can you fix the merge conflicts ASAP svp?

**Pull request merged**